src/app/pages/OpenWalletPage/Features/FromLedger/__tests__/index.test.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+import { render, act, screen } from '@testing-library/react'
+import { FromLedger } from '..'
+import { MemoryRouter } from 'react-router-dom'
+
+jest.mock('react-redux', () => ({
+  useSelector: jest.fn(),
+  useDispatch: jest.fn(),
+}))
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => jest.fn(),
+}))
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => jest.fn(),
+}))
+
+jest.mock('../../../../../lib/ledger', () => ({
+  ...jest.requireActual('../../../../../lib/ledger'),
+  // Throws BLE not supported
+  canAccessBle: () => jest.fn().mockReturnValue(false),
```

Comment:
this should mimic promise mockResolvedValue

Reply:
I think it's because you are wrapping jest.fn() with arrow function 
` canAccessBle: jest.fn().mockResolvedValue(false),` should work

Reply:
This minor issue got lost in comments too

Reply:
Right 👍 fixed